website/cue/reference/configuration.cue

```diff
@@ -277,7 +289,7 @@ configuration: {
 						description: """
 							Controls which tags should be included with the `vector_component_sent_events_total` and
 							`vector_component_sent_event_bytes_total` metrics.
-						"""
+							"""
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏ I noticed this the other day too. Thanks for fixing!